README-CN.md

```diff
@@ -0,0 +1,73 @@
+# terraform-alicloud-emr-hadoop
+---
+
+本 Module 用于在阿里云的 VPC 下创建一个[E-MapReduce](https://help.aliyun.com/document_detail/28068.html). 
+
+本 Module 支持创建以下资源:
+
+* [E-MapReduce集群实例](https://www.terraform.io/docs/providers/alicloud/r/emr_cluster.html)
+
+## Terraform 版本
+
+本模板要求使用版本 Terraform 0.12.
+
+用法
+-----
+
+
+```hcl
+
+data "alicloud_emr_main_versions" "default" {
+  cluster_type = ["HADOOP"]
+}
+
+data "alicloud_vpcs" "default" {
+  is_default = true
+}
+
+data "alicloud_vswitches" "all" {
+  vpc_id = data.alicloud_vpcs.default.ids.0
+}
+
+module "security_group" {
+  source  = "alibaba/security-group/alicloud"
+  vpc_id  = data.alicloud_vpcs.default.ids.0
+  version = "~> 2.0"
+}
+
+module "emr-hadoop" {
+  source = "terraform-alicloud-modules/emr-hadoop/alicloud"
+  emr_version = data.alicloud_emr_main_versions.default.main_versions.0.emr_version
+  option_software_list = ["HBASE","PRESTO",]
+  charge_type = "PostPaid"
+  vswitch_id = data.alicloud_vswitches.all.ids.0
+  security_group_id = module.security_group.this_security_group_id
+  high_availability_enable = true
+  is_open_public_ip = true
+  ssh_enable = true
+  master_pwd = "YourPassword123!"
+}
+```
+
+## 示例
+
+* [E-MapReduce HADOOP集群示例](https://github.com/terraform-alicloud-modules/terraform-alicloud-emr-hadoop/tree/master/example)
+
+## 注意
+
+* 本 Module 使用的 AccessKey 和 SecretKey 可以直接从 `profile` 和 `shared_credentials_file` 中获取。如果未设置，可通过下载安装 [aliyun-cli](https://github.com/aliyun/aliyun-cli#installation) 后进行配置.
+
+作者
+-------
+Created and maintained by He Guimin(@xiaozhu36, heguimin36@163.com)
+Created and maintained by Qi yinfei(@yfqi, qiyf_shadow@yeah.net)
```

Comment:
Created and maintained by Qi yinfei(@yfqi, qiyf_shadow@yeah.net), He Guimin(@xiaozhu36, heguimin36@163.com)